templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Login</h2>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
```

Comment:
Displaying a generic error message like 'Invalid credentials' is a good practice as it doesn't reveal whether the username or password was incorrect, enhancing security.